packages/@react-spectrum/s2/src/Disclosure.tsx

```diff
@@ -49,7 +49,14 @@ const disclosure = style({
   },
   borderBottomWidth: {
     default: 0,
-    ':last-child': {
+    isInGroup: {
+      default: 0,
+      ':last-child': {
+        default: 1,
+        isQuiet: 0
+      }
+    },
+    isNotInGroup: {
```

Comment:
given these are mutually exclusive, should `isNotInGroup` be the default case?
then we don't need the extra variable

Reply:
Ahh thanks, kew I was overthinking that